src/pages/DistributionsPage/DistributionsPage.tsx

```diff
@@ -99,18 +99,39 @@ export function DistributionsPage() {
     );
   };
 
+  const deletedUsers: Gift[] = uniqBy(
```

Comment:
should I memoize this? I tried wrapping in useMemo but got an error about the order of rendering changing...

Reply:
yeah, you just need to invoke the hook unconditionally in order to avoid the error